src/scripts/main.js

```diff
@@ -2,6 +2,29 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const body = document.querySelector('body');
+
+  const message = document.createElement('div');
+  const titleElement = document.createElement('h2');
+  const descriptionElement = document.createElement('p');
+
+  titleElement.textContent = title;
+  descriptionElement.textContent = description;
+
+  message.classList.add('notification');
+  message.classList.add(`${type}`);
+  titleElement.classList.add('title');
+
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  setTimeout(() => {
+    body.removeChild(message);
+  }, 2000);
+
+  body.insertBefore(message, body.lastElementChild);
```

Comment:
Using `body.insertBefore(message, body.lastElementChild);` may not work as intended if the body has no children or if the last child is not the intended insertion point. Consider using `body.appendChild(message);` to append the message to the end of the body.